documentation/en/faqs.md

```diff
@@ -0,0 +1,134 @@
+# Frequently Asked Questions
+
+Here are some FAQs concerning the Lotus implementation and participation in 
+Testnet.
+For questions concerning the broader Filecoin project, please 
+go [here](https://filecoin.io/faqs/).
+
+## Introduction to Lotus
+
+#### What is Lotus?
+
+Lotus is an implementation of the **Filecoin Distributed Storage Network**, written in Go. 
+It is designed to be modular and interoperable with any other implementation of the Filecoin Protocol.
+More information about Lotus can be found [here](https://lotu.sh/).
+
+#### What are the components of Lotus?
+
+Lotus is composed of two separate pieces that can talk to each other:
+
+The Lotus Node can sync the blockchain, validating all blocks, transfers, and deals
+along the way. It can also facilitate the creation of new storage deals. If you are not 
+interested in providing your own storage to the network, and do not want to produce blocks
+yourself, then the Lotus Node is all you need!
+
+The Lotus Storage Miner does everything you need for the registration of storage, and the
+production of new blocks. The Lotus Storage Miner communicates with the network
+by talking to a Lotus Node over the JSON-RPC API.
+
+## Setting up a Lotus Node
+
+#### How do I set up a Lotus Node?
+
+Follow the instructions found [here](https://docs.lotu.sh/en+getting-started).
+
+#### Where can I get the latest version of Lotus?
+
+Download the binary tagged as the `Latest Release` from the
+ [Lotus Github repo](https://github.com/filecoin-project/lotus/releases).
+ 
+#### What operating systems can Lotus run on?
+
+Lotus can build and run on most Linux and MacOS systems with at least 
+8GB of RAM. Windows is not yet supported.
+
+#### How can I update to the latest version of Lotus?
+
+To update Lotus, follow the instructions [here](https://lotu.sh/en+updating-lotus).
+
+#### How do I prepare a fresh installation of Lotus?
+
+Stop the Lotus daemon, and delete all related files, including sealed and chain data by 
+running `rm ~/.lotus ~/.lotusstorage`.
+
+Then, install Lotus afresh by following the instructions 
+found [here](https://docs.lotu.sh/en+getting-started).
+
+## Interacting with a Lotus Node
+
+#### How can I communicate with a Lotus Node?
+
+Lotus Nodes have a command-line interface, as well as a JSON-RPC API.
+
+#### What are the commands I can send using the command-line interface? 
+
+The command-line interface is self-documenting, try running `lotus --help` from the `lotus` home 
+directory for more.
```

Comment:
Nit: you may want to check prettier or the way you're doing word wrapping, it will look weird no matter what markdown renderer ends up picking this up.